src/app/auto-complete/auto-complete.component.css

```diff
@@ -6,7 +6,6 @@ a {
   font-family: Arial, sans-serif;
   font-size: 17px;
   margin-left: 2.4%;
-  display: block;
```

Comment:
Please add this line back, why are you reverting changes made in #818?

Reply:
@aashutoshrathi But I didn't edit the 'auto-complete.component.css'.

Reply:
Oh then rebase, please 😅